### DISCUSSION TITLE
分享 快捷链接/搜索引擎

### DISCUSSION THREAD

**Discussion by GH01** [hcfyapp/crx-selection-translate#945] [讨论]
https://github.com/lmk123/crx-selection-translate/issues/940#issuecomment-808555093
[快捷链接]将增强！
迫不及待开起这个讨论，欢迎大家来献宝！

附上相关讨论 #879 涉及：
跳转链接
快捷链接
搜索引擎
右键菜单

**Comment by lmk123** [comment]
我把这个帖子写进快捷链接的说明里了，以后这个帖子就专门用来分享快捷链接。

顺便分享一个我自己在用的：

名称：豆瓣电影
链接：https://search.douban.com/movie/subject_search?search_text=%s&cat=1002

**Comment by miask** [comment]
日语 沪江小D 
链接 https://dict.hjenglish.com/notfound/jp/jc/%s

**Comment by tianjianchn** [comment]
有道词典：https://dict.youdao.com/search?q=%s

**Comment by A5Kush** [comment]
油管：
https://www.youtube.com/results?search_query=%s

Bilibili:
https://search.bilibili.com/all?keyword=%s

谷歌学术:  
https://scholar.google.com/scholar?hl=en&as_sdt=0%2C48&q=%s

Bandcamp:
https://bandcamp.com/search?q=%s

**Comment by NatsuGaOwatta** [comment]
- Collins English Dictionary: https://www.collinsdictionary.com/dictionary/english/%s
- Weblio辞書: https://www.weblio.jp/content/%s
- goo辞書: https://dictionary.goo.ne.jp/srch/all/%s/m0u/
- コトバンク: https://kotobank.jp/gs/?q=%s
可惜常用的 MOJi辞書 没法一个 GET 请求到单词页，参考 [mojidict/engine.ts](https://github.com/crimx/ext-saladict/blob/v7.20.0/src/components/dictionaries/mojidict/engine.ts)

**Comment by ZLFboy** [comment]
有大佬分享下CNKI翻译助手的快捷链接怎么设置吗？
https://dict.cnki.net/index

**Reply by lmk123** [reply]
我试了一下，CNKI 翻译助手在翻译文本时不会把内容写进地址栏，所以做不了快捷链接

**Reply by A5Kush** [reply]
> 我试了一下，CNKI 翻译助手在翻译文本时不会把内容写进地址栏，所以做不了快捷链接我试了下，CNKI 翻译助手在翻译中文本时不会把内容写进地址栏，所以做不到快捷链接

请问 快捷链接功能 有可能增加便捷调整顺序的办法吗？例如拖动或者类似选择翻译排列那样？辛苦了！

**Comment by ZLFboy** [comment]
好的，感谢


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2022年12月7日 12:13 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [hcfyapp/crx-selection-translate] 分享 快捷链接/搜索引擎 (Discussion #945) |

我试了一下，CNKI 翻译助手在翻译文本时不会把内容写进地址栏，所以做不了快捷链接

**Comment by Wilyiam** [comment]
汉典：https://www.zdic.net/hans/%s

**Comment by meihus** [comment]
将复制的文字转化为二维码：https://haoip.cn/qrcode?t=%s
                 QR码生成与识别：https://guokai.dev/qrcode/generator?text=%s
                 牛津词典：https://www.oxfordlearnersdictionaries.com/definition/english/%s

**Comment by CaptureVibes** [comment]
牛津词典
https://www.oxfordlearnersdictionaries.com/us/definition/english/%s_1?q=%s

**Reply by ironald** [reply]
好像不行了

**Reply by a-voyager** [reply]
https://www.oxfordlearnersdictionaries.com/us/definition/english/%s

**Comment by wwillxu** [comment]
竟然还没有朗文词典
https://www.ldoceonline.com/dictionary/%s

**Reply by meihus** [reply]
404啊

**Reply by wwillxu** [reply]
啊国内没法用么😂

**Reply by meihus** [reply]
开了全局也不行

**Reply by wwillxu** [reply]
会不会是DNS的问题

**Reply by meihus** [reply]
https://www.ldoceonline.com/我可以打开朗文的官网

**Reply by wwillxu** [reply]
那就没问题才对呀

**Comment by xjacks** [comment]
追加一个法语字典的查询：
名称：法语助手
链接：https://www.frdic.com/dicts/fr/%s

**Comment by dilligent** [comment]
Merriam-Webster  
https://www.merriam-webster.com/dictionary/%s

**Comment by ntol** [comment]
名称：Metaso
链接：https://metaso.cn/?q=%s

该搜索分为[简洁]、[深入]、[研究]三种搜索强度，根据自身需求可在搜索页面左侧功能栏中，点击那个长得像放大镜，实则是[主页]的超链图标，以修改搜索强度。

**Comment by Sev73n** [comment]
名称:
必应搜索
链接:
https://cn.bing.com/search?q=%s

**Comment by McLaren404** [comment]
名称：
uniprot
https://www.uniprot.org/uniprotkb?query=%s

**Comment by guaijiu9** [comment]
個人使用體驗比較好用的翻譯 (已設定好格式)：

https://www.wordreference.com/enzh/%s

https://glosbe.com/en/zh/%s

https://www.linguee.com/english-chinese/search?source=english&query=%s

https://papago.naver.com/?sk=auto&tk=zh-TW&st=%s

https://www.bing.com/translator?from=auto&to=zh-TW&text=%s

https://www.deepl.com/en/translator#en/zh-TW/%s%3F

https://translate.google.com/?sl=en&tl=zh-TW&text=%s&op=translate

**Comment by dilligent** [comment]
Cambridge Dictionary

https://dictionary.cambridge.org/dictionary/english/%s

**Comment by vipouno** [comment]
名称：
闲鱼

链接：
https://www.goofish.com/search?q=%s
